### ISSUE TITLE
Ember creating new blank artwalk/favorite when I click back

### ISSUE THREAD

**Issue by kiraamaa** [ga-wdi-boston/capstone-project#410]
It's the same problem we were having in class during the ember-resources code-along where clicking `Back to favorites` results in a new favorite being made. It doesn't make one when cancel is clicked, but creates one when save is clicked and when the `#link-to 'favorites` is clicked. The input field is cleared with rollbackAttribute.

**Comment by kiraamaa** [comment]
Did not solve; if user hits back button in browser blank object is still created.

**Comment by raq929** [comment]
What's the status on this?

**Comment by raq929** [comment]
Related to #419

**Comment by kiraamaa** [comment]
Used Caleb's fix from #419 to inhibit creation of new artwalk by adding the following code to my artwalks/new/route.js file. Now having same issue but with edit. Although new "edited" title is not persisted in the back end, there is still a user-facing bug that seemingly changes the artwalk name until the page is refreshed.
``` js
willTransition () {
     let store = this.get('store');
     store.peekAll('artwalk').forEach(function (artwalk) {
       if (artwalk.get('isNew') && artwalk.get('hasDirtyAttributes')) {
         artwalk.rollbackAttributes();
       }
     });
     return true;
   },
```

**Comment by kiraamaa** [comment]
Resolved by adding this code to favorites/new/route.js and similar code minus `artwalk.get('isNew') &&` to favorite/edit/route.js and artwalk/edit/route.js.
```js
willTransition () {
     let store = this.get('store');
     store.peekAll('favorite').forEach(function (favorite) {
       if (favorite.get('isNew') && favorite.get('hasDirtyAttributes')) {
         favorite.rollbackAttributes();
       }
     });
     return true;
   },
```
